composer/trainer/trainer.py

```diff
@@ -711,8 +712,8 @@ class Trainer:
             .. seealso:: :mod:`composer.utils.reproducibility` for more details on reproducibility.
         dist_timeout (float, optional): Timeout, in seconds, for initializing the distributed process group.
             (default: ``15.0``)
-        ddp_sync_strategy (str | DDPSyncStrategy, optional): The strategy to use for synchronizing gradients.
-            Leave unset to let the trainer auto-configure this. See :class:`.DDPSyncStrategy`
+        sync_strategy (str | SyncStrategy, optional): The strategy to use for synchronizing gradients.
```

Comment:
Seems like if DDP is not used, this is a no-op, so perhaps keep the name?

Reply:
Yeah I think it's a No-op right now, but there is some kind of FSDP sync strategy thing that I didn't have time to integrate (I think its necessary for cpu_offload). If it's better optics I can revert this rename and then just keep in mind we may have a `fsdp_sync_strategy` in the future?

Reply:
Yeah let's do that rather than introduce something breaking now for no functionality